doc/source/whatsnew/v1.0.2.rst

```diff
@@ -25,8 +25,9 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 
--
--
+**I/O**
```

Comment:
Thanks @ArtificialQualia 

This should be
```
I/O
^^^
```

Reply:
@MarcoGorelli I've updated that to the correct format.

Reply:
The original was actually fine, to be consistent with the v1.0.1 file

(sorry for the conflicting comments)

Reply:
My mistake, sorry, I'd taken that from v1.1.0 where we go back to
```
I/O
^^^
```
, will be more careful in the future

Reply:
@MarcoGorelli no problem! It's also inconsistent between the different files .. (for the bug fix releases, the list of bugs is much smaller, so less worth it to make subsections. At least, that is what we just did for the v1.0.1 file)

Reply:
Reverted to original format